roles/httpd/templates/foreman-ssl-vhost.conf.j2

```diff
@@ -0,0 +1,104 @@
+<VirtualHost *:443>
+ServerName {{ ansible_fqdn }}
+
+  ## Load additional static includes
+  IncludeOptional "/etc/httpd/conf.d/05-foreman-ssl.d/*.conf"
+
+  ## Logging
+  ErrorLog "/var/log/httpd/foreman-ssl_error_ssl.log"
+  ServerSignature Off
+  CustomLog "/var/log/httpd/foreman-ssl_access_ssl.log" combined
+
+  ## Request header rules
+  ## as per http://httpd.apache.org/docs/2.4/mod/mod_headers.html#requestheader
+  RequestHeader set X_FORWARDED_PROTO "https"
+  RequestHeader set SSL_CLIENT_S_DN "%{SSL_CLIENT_S_DN}s"
+  RequestHeader set SSL_CLIENT_CERT "%{SSL_CLIENT_CERT}s"
+  RequestHeader set SSL_CLIENT_VERIFY "%{SSL_CLIENT_VERIFY}s"
+  RequestHeader unset REMOTE_USER
+  RequestHeader unset REMOTE_USER_EMAIL
+  RequestHeader unset REMOTE_USER_FIRSTNAME
+  RequestHeader unset REMOTE_USER_LASTNAME
+  RequestHeader unset REMOTE_USER_GROUPS
+
+  # SSL Proxy directives
+  SSLProxyEngine On
+
+  ProxyPass /pulp_ansible/galaxy/ {{ pulp_api_backend }}/pulp_ansible/galaxy/
+  ProxyPassReverse /pulp_ansible/galaxy/ {{ pulp_api_backend }}/pulp_ansible/galaxy/
+
+  <Location "/pulpcore_registry/v2/">
+    RequestHeader unset REMOTE_USER
+    RequestHeader set REMOTE_USER "%{SSL_CLIENT_S_DN_CN}s" env=SSL_CLIENT_S_DN_CN
+    ProxyPass {{ pulp_api_backend }}/v2/
+    ProxyPassReverse {{ pulp_api_backend }}/v2/
+  </Location>
+
+  ProxyPass /pulp/container/ {{ pulp_content_backend }}/pulp/container/
+  ProxyPassReverse /pulp/container/ {{ pulp_content_backend }}/pulp/container/
+
+  <Location "/pulp/isos">
```

Comment:
Reminds me that this doesn't hurt anything, but we should track removing it.

Reply:
yeah, I literally copied over the 05-ssl.conf from an existing machine as a template :see_no_evil: